srcpkgs/parprouted/files/parprouted/run

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+if [ -r ./conf ]; then
+    . ./conf
+else
+    echo "Add conf file!"
+    exit 1
+fi
+
+if [ -z "${EXIT_IF}" ] || [ -z "${IN_IF}" ]; then
+    echo "Add data in conf file!"
+    exit 1
+fi
+
```

Comment:
Not a fan of this being part of the run script.
```
#!/bin/sh
[ -r ./conf ] && . ./conf
exec paperouted ${OPTS}
```

should be enough, let the service fail and don't add logic to the run script that might change.

Reply:
yeah ok